agent/doc/UML/agent_device_management_design.puml

```diff
@@ -154,4 +154,45 @@ LinuxScanner -u-|> EnvCapabilityScanner
 MacOSScanner -u-|> EnvCapabilityScanner
 
 EnvCapabilityScanner .. EnvCapability: produce
+@enduml
+
+@startuml agent_capability_design
+
+class AgentManagementService {
+    EnvInfo envInfo;
+    List<ServiceCapability> serviceCapability;
+    detectAndRegisterServiceCapability(ServiceCapability)
+}
+AgentManagementService -r-> EnvCapabilityDiscoveryService
+class TestRunnerConfig
+class TestRunner{
+    ServiceCapability serviceCapability;
+}
+class DeviceDriverConfig
+class DeviceDriver{
+    ServiceCapability serviceCapability;
+}
+TestRunnerConfig ..> TestRunner : create
+TestRunnerConfig -r-> AgentManagementService
+DeviceDriverConfig ..> DeviceDriver : create
+DeviceDriverConfig -r-> AgentManagementService
+TestRunner --> ServiceCapability
+DeviceDriver --> ServiceCapability
+
+class ServiceCapability{
+    +String serviceName
+    +enum serviceType
+    +Map<CapabilityKeyword, String miniVersion > reliantEnvCapabilities
+    +Map<CapabilityKeyword, boolean isReady> detectDetails
+    +boolean isReady
+}
+ServiceCapability *--> serviceType
+enum serviceType{
```

Comment:
Could you add a comment to describe what's the meaning of this service type? Where will this be used?